docs/concepts/services/fetch.md

```diff
@@ -26,7 +26,7 @@ pub type Binary = Result<Vec<u8>, Error>;
 Here is what a typical GET request will look like:
 ```rust
 use yew::format::Nothing;
-use yew_services::fetch::Request;
+use yew::services::fetch::Request;
```

Comment:
These changes are wrong, since the layout was recently changed and the services are in a separate crate now: <https://github.com/yewstack/yew/tree/master/packages/yew-services>
Please note that you're editing the documentation for the 'master' branch which contains changes that haven't been published yet.

Reply:
I definitely missed that.
Shall I remove the yew_services changes and still add the wording changes here in master?

Reply:
Yeah, I think the wording changes are great!